packages/babel-core/register.js

```diff
@@ -1,3 +0,0 @@
-/* eslint max-len: 0 */
-// TODO: eventually deprecate this console.trace("use the `babel-register` package instead of `babel-core/register`");
-module.exports = require("babel-register");
```

Comment:
Could we add an error as in [packages/babel/index.js](https://github.com/babel/babel/blob/master/packages/babel/index.js#L1)?

I don't want people to loose time about the resolution of this issue, what do you think?